pydantic/typing.py

```diff
@@ -399,7 +399,10 @@ def resolve_annotations(raw_annotations: Dict[str, Type[Any]], module_name: Opti
             else:
                 value = ForwardRef(value, is_argument=False)
         try:
-            value = _eval_type(value, base_globals, None)
+            if sys.version_info > (3, 13):
```

Comment:
```suggestion
            if sys.version_info[:2] >= (3, 13):
```

Reply:
I don't think I think we use the pattern I've suggested for almost all of our version info checks...